build.sh

```diff
@@ -24,6 +24,7 @@ ${DOCKER} run --rm \
     -v "${PWD}/cri-dockerd-${CRI_DOCKERD_VERSION}-${ARCH}.LICENSE:/home/build/cri-dockerd.license:ro" \
     -v "${PWD}/sshd.pam:/home/build/sshd.pam:ro" \
     $(env | grep ^LIMA_ | xargs -n 1 printf -- '-e %s ') \
+    -e "LIMA_INSTALL_CGROUP_CONF=${LIMA_INSTALL_CGROUP_CONF}" \
```

Comment:
This line is not needed; the line above is exporting all `LIMA_*` environment variables into the container automatically. The 3 variables below are exceptions because they are renaming variables that don't start with `LIMA_*` on the host.